ci_framework/plugins/action/ci_make.py

```diff
@@ -181,21 +182,21 @@ def extract_env(self, task_vars):
 
     def run(self, tmp=None, task_vars=None):
         super(ActionModule, self).run(tmp, task_vars)
-        module_args = self._task.args.copy()
 
+        task_args = self._task.args.copy()
```

Comment:
not exactly replacing since you're keeping blank lines - intended?

Reply:
that one hasn't change, I make each conversion individually further in the code, as I need the original copy